src/components/measureLanding/measureList/MeasureList.tsx

```diff
@@ -446,7 +450,11 @@ export default function MeasureList(props: {
     return true;
   };
   const handlePopOverOpen = useCallback(
-    async (selected: Measure, event: React.MouseEvent<HTMLButtonElement>) => {
+    async (
+      selected: Measure,
+      event: React.MouseEvent<HTMLButtonElement>,
+      qdmExportFlag: boolean
```

Comment:
why was it not caught by tests? is there no test around version option?

Reply:
The strange thing is there are tests for export option and all these tests are passing

Reply:
In the UI whenever we access the featureFlags we get first values as false and then it updates to real values so in the tanstack is only considering the false values because it is not being updated. but whereas in the test case we are manually sending all the correct feature flags in the start itself so it is getting correct values and it is passing test case

Reply:
that's interesting. maybe our test is not doing what it supposed to do.